lib/src/sdk.test.ts

```diff
@@ -397,3 +411,88 @@ describe("fund", () => {
     );
   });
 });
+
+describe("fundDex", () => {
```

Comment:
Follow our conversation in the slack channel
```suggestion
describe("tokenApproval", () => {
```